frontend/src/components/SupportingDocumentUpload.vue

```diff
@@ -419,17 +401,17 @@ export default {
       this.isProcessing = true;
       try {
         await this.processDocumentFileDelete();
-        const newFilesAdded = this.uploadedSupportingDocuments.filter((el) => !!el.id);
-        if (newFilesAdded.length > 0) {
+        const newFilesAdded = this.uploadedSupportingDocuments?.filter((el) => !!el.id);
+        if (newFilesAdded?.length > 0) {
```

Comment:
Could switch to isEmpty(). But the code is already a mix so fine as is as well.

Reply:
I've updated all .length > 0 to use isEmpty in this file